pcsx2/R3000A.cpp

```diff
@@ -172,9 +172,10 @@ static __fi void _psxTestInterrupts()
 	IopTestEvent(IopEvt_SIF0,		sif0Interrupt);	// SIF0
 	IopTestEvent(IopEvt_SIF1,		sif1Interrupt);	// SIF1
 	IopTestEvent(IopEvt_SIF2,		sif2Interrupt);	// SIF2
-#ifndef SIO_INLINE_IRQS
-	IopTestEvent(IopEvt_SIO,		sioInterrupt);
-#endif
+	// Originally controlled by a preprocessor define, now PSX dependent.
+	if (psxMode) {
+		IopTestEvent(IopEvt_SIO, sioInterruptR);
```

Comment:
All the conditional checks at this file use this format:

```
if(condition)
{
}
```

not 

```
if(condition) {
}
```

I'd prefer to keep the style which is more common. Though I think it would be better if braces are not used for single line statements.